lib/modules/manager/bazel-module/extract.ts

```diff
@@ -0,0 +1,20 @@
+import { logger } from '../../../logger';
+import { LooseArray } from '../../../util/schema-utils';
+import type { PackageFileContent } from '../types';
+import { ToBazelDep } from './bazel-dep';
+import { parse } from './parser';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
```

Comment:
```suggestion
  fileName: string
```
we usually log it with that name